angular-workspace/projects/ni/nimble-angular/table-column/enum-text/nimble-mapping-text.directive.ts

```diff
@@ -0,0 +1,32 @@
+import { Directive, ElementRef, Input, Renderer2 } from '@angular/core';
+import { type MappingText, mappingTextTag } from '@ni/nimble-components/dist/esm/mapping/text';
```

Comment:
> Both of these are in a single moduleBoth of these are in a single module

Pretty confident we have not done that for any components in nimble. They all get there own module. If you found a location that was exporting multiple components from the same module we should probably fix that to be consistent with the rest.

Reply:
May be useful to capture something in CONTRIBUTING to enforce consistency